<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HOME" failures="0" tests="7" name="com.darkuros.selenium.tests.LoginTest" time="68.908" errors="1" timestamp="2025-07-01T21:57:03 IST" skipped="1">
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="LoginWithValidCredentials" time="11.171">
    <skipped/>
  </testcase> <!-- LoginWithValidCredentials -->
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="LoginWithInvalidUserEmail" time="8.837"/>
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="LoginWithValidCredentials" time="12.532">
    <error message="Expected condition failed: waiting for url to be &quot;https://rahulshettyacademy.com/client/dashboard/dash&quot;. Current url: &quot;https://rahulshettyacademy.com/client/auth/login&quot; (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.33.0&#039;, revision: &#039;2c6aaad03a&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.7&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 140.0.2, moz:accessibilityChecks: false, moz:buildID: 20250627085530, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 18544, moz:profile: C:\Users\ABHIN~1.HOM\AppDat..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Session ID: 9eda2d37-20c9-41f2-bb11-2de8c46eca20" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for url to be "https://rahulshettyacademy.com/client/dashboard/dash". Current url: "https://rahulshettyacademy.com/client/auth/login" (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: '4.33.0', revision: '2c6aaad03a'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.7'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 140.0.2, moz:accessibilityChecks: false, moz:buildID: 20250627085530, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 18544, moz:profile: C:\Users\ABHIN~1.HOM\AppDat..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Session ID: 9eda2d37-20c9-41f2-bb11-2de8c46eca20
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at com.darkuros.selenium.pageobjects.LandingPage.isHomePageDisplayed(LandingPage.java:21)
at com.darkuros.selenium.tests.LoginTest.LoginWithValidCredentials(LoginTest.java:28)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- LoginWithValidCredentials -->
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="isRegisterLinkWorking" time="7.441"/>
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="isForgotPasswordLinkWorking" time="8.599"/>
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="LoginWithInvalidCredentials" time="9.687"/>
  <testcase classname="com.darkuros.selenium.tests.LoginTest" name="LoginWithInvalidPassword" time="10.641"/>
</testsuite> <!-- com.darkuros.selenium.tests.LoginTest -->
